libs/CloudscalerLibcloud/CloudscalerLibcloud/compute/drivers/libvirt_driver.py

```diff
@@ -337,15 +337,18 @@ def _create_disk(self, vm_id, disksize, image, disk_role='base'):
         volumeid = self.getVolumeId(vdiskguid=vdiskguid, edgeclient=edgeclient, name=diskname)
         return OpenvStorageVolume(id=volumeid, name='Bootdisk', size=disksize, driver=self), edgeclient
 
-    def create_volume(self, size, name):
+    def create_volume(self, size, name, data=True):
         volumes = [{'name': name, 'size': size, 'dev': ''}]
-        return self.create_volumes(volumes)[0]
-
-    def create_volumes(self, volumes):
+        return self.create_volumes(volumes, data=data)[0]
```

Comment:
it would probably be better to include the data boolean flag inside the disk dict